src/main/java/io/github/mivek/parser/AbstractParser.java

```diff
@@ -170,14 +158,14 @@ protected void parseRMK(final AbstractWeatherContainer pContainer, final String[
      * @param pCode the string to parse
      * @return a array of tokens
      */
-    protected String[] tokenize(final String pCode) {
+    String[] tokenize(final String pCode) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/mivek/MetarParser/pullRequest?prid=4593741)